docs/source/man.j2

```diff
@@ -0,0 +1,816 @@
+{{ QPC_VAR_PROGRAM_NAME }}
+===
+
+Name
+----
+
+{{ QPC_VAR_PROGRAM_NAME }} - Inspect and report on product entitlement metadata from various sources, including networks and systems management solutions.
+
+
+Synopsis
+--------
+
+``{{ QPC_VAR_PROGRAM_NAME }} command subcommand [options]``
+
+Description
+-----------
+
+The {{ QPC_VAR_PROJECT }} tool, accessed through the ``{{ QPC_VAR_PROGRAM_NAME }}`` command, is an inspection and reporting tool. It is designed to identify environment data, or *facts*, such as the number of physical and virtual systems on a network, their operating systems, and other configuration data. In addition, it is designed to identify and report more detailed facts for some versions of key Red Hat packages and products for the Linux based IT resources in that network. The ability to inspect the software and systems that are running on your network improves your ability to understand and report on your usage. Ultimately, this inspection and reporting process is part of the larger system administration task of managing your inventories.
```

Comment:
I understand quipucords/discovery are not actual products, but for the purpose of this document it could be clearer if we use this nomenclature (`QPC_VAR_PRODUCT`). Thoughts @infinitewarp @nicolearagao ?

Reply:
or maybe `QPC_VAR_SERVER_NAME` to be more explicit 🤔

Reply:
`QPC_VAR_PROJECT` resolves to `Quipucords`, not `qpc`.

I believe the current implementation by @nicolearagao matches the behavior of the existing implementation. Do we really want to change that here? I think that should be a separate discussion and PR if so. Changing that value would fall under @bluemorpho-mpw's expertise.

Reply:
++ to discussion it further later on and modifying it with a supplemental pr.